shared/src/business/entities/cases/PublicCase.js

```diff
@@ -104,6 +105,12 @@ const publicCaseSchema = {
     .description('Party type of the case petitioner.'),
   petitioners: joi.array().items(PublicContact.VALIDATION_RULES).required(),
   receivedAt: JoiValidationConstants.ISO_DATE.optional(),
+  status: JoiValidationConstants.STRING.valid(
+    ...Object.values(CASE_STATUS_TYPES),
+  )
+    .optional()
+    .meta({ tags: ['Restricted'] })
```

Comment:
Hello! Just curious to know what this `[Restricted]` tag does!

Reply:
@matthopson can you answer this?

Reply:
This was pulled over from the other definitions of this field on Case. I'm not 100% sure, as I couldn't find where it was being used, but I thought these would be restricted fields on a blocked case - which makes this a good question to try and answer for sure.

Reply:
Eric said it was used for document generation and doesn't change anything. :)